packages/core/server/admin-api/services/query-layer-decorator.ts

```diff
@@ -250,6 +250,76 @@ const decorator = (service: IDecoratedService) => ({
     // Eventually delete the entity.
     return service.delete.call(this, uid, entityId);
   },
+
+  // eslint-disable-next-line max-len
+  async clone(uid: Common.UID.ContentType, cloneId: number, params?: IDecoratedServiceOptions<{ url_alias: number }>) {
+    const hasWT = isContentTypeEnabled(uid);
+    if (!hasWT) {
+      return service.clone.call(this, uid, cloneId, params);
+    }
+
+    // Manually fetch the entity to clone
+    const entityToClone = await service.findOne.call(this, uid, cloneId, { populate: ['url_alias'] });
+
+    if (!entityToClone) {
+      throw new Error('Entity to clone not found');
+    }
+
+    // Remove the id and url_alias from the entity to clone
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    const { id, url_alias, ...clonedData } = entityToClone;
+
+    // Create the cloned entity
+    const clonedEntity = await service.create.call(this, uid, { data: clonedData });
```

Comment:
You likely need to pass the `params` as the third parameter of the `.call` function.

If you do that, you don't need the `clonedData`, which means you also don't need to manually fetch the `entityToClone`. That saves on the amount of DB requests! 👍

Also it would probably be better to call the `clone` service here, instead of the `create` service. That way we can make sure we don't change any core functionality from Strapi.

Reply:
This has been solved.